src/models/course.ts

```diff
@@ -23,13 +26,14 @@ export const courseSchema = new dynamoose.Schema(
 );
 
 export interface CourseInterface extends Document {
-    id: string;
+    id: string
     name: string;
     desiredGrade: number | FractionInterface;
-    courseItems: CourseItemInterface[];
-    owner: string;
+    courseItems?: Set<CourseItemInterface>;
+    owner: string
```

Comment:
I don't think it's required but for consistency maybe all of the lines should end with semicolons?

Reply:
Aah I totally mistyped here, thanks for catching this, Amanda!